apps/docs/guide/deploying/self-hosted.md

```diff
@@ -0,0 +1,164 @@
+# Self-hosted deployment
+
+This method of deployment allows the developer to run his own infrastructure to host the application node. The developer will need the following:
+
+1. a server connected to the internet (i.e. cloud server);
+2. a postgres database reachable from the server;
+3. a web3 provider;
+4. a wallet with funds of the [base layer of choice](./supported-networks.md).
+
+## Deploying
+
+The self-hosted deployment process can be initiated by the `sunodo deploy --hosting self-hosted` command.
+
+That command will show the Cartesi machine hash and build a Docker image containing the Cartesi rollups node and the Cartesi machine:
+
+```shell
+$ sunodo deploy --hosting self-hosted
+? Cartesi machine templateHash 0xc87999b8a93609268b10de25f2e49d35f80fad92813310edc585ed644a9805d3
+? Application node Docker image sha256:5c355a9bddc92aa08987f395a257a0b32a51552c969eb161386e46f9380ea2ac
+```
+
+The image produced, in this example `sha256:5c355a...0ea2ac`, can be tagged and pushed to a Docker registry of choice, like Docker Hub, or any other private or public registry.
+
+The next step is to deploy the smart contract for the application. The deploy command will redirect the developer to a web application, where he can use a wallet to deploy the required smart contracts.
+
+![deploy self-hosted](./self-hosted-web.png)
+
+## Wallet
+
+The developer will need a wallet that will be used by the Cartesi rollups node to submit transactions to the base layer. This wallet should ideally be used exclusively for that purpose.
+
+A new wallet can be created using a tool like [cast](https://book.getfoundry.sh/reference/cast/cast-wallet-new-mnemonic). A wallet managed by [AWS KMS](https://aws.amazon.com/blogs/database/part1-use-aws-kms-to-securely-manage-ethereum-accounts/) is also supported for an increased level of security.
+
+::: warning
+The deploy web application need the wallet PUBLIC ADDRESS only!
```

Comment:
```suggestion
The deploy web application needs the wallet PUBLIC ADDRESS only!
```
This sentence is a bit weird to read. I guess "deploy web application" is the name of the web app but sounds a bit strange. Maybe "deployment web application" or "web app for deployment"